src/Card/CardFooter.jsx

```diff
@@ -1,20 +1,37 @@
 import React, { useContext } from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
+import Skeleton from 'react-loading-skeleton';
 import CardContext from './CardContext';
 
+const IS_LOADING_HEIGHT_VALUE = 18;
+
 const CardFooter = React.forwardRef(({
   children,
   className,
   isStacked,
   textElement,
+  skeletonHeight,
+  skeletonWidth,
   orientation,
 }, ref) => {
-  const { orientation: cardOrientation } = useContext(CardContext);
+  const { orientation: cardOrientation, isLoading } = useContext(CardContext);
   const footerOrientation = orientation || cardOrientation;
   const wrapperClassName = `pgn__card-footer ${footerOrientation}${isStacked ? '-stacked' : ''}`;
   const textElementClassName = `pgn__card-footer-text ${footerOrientation}${isStacked ? '-stacked' : ''}`;
 
+  if (isLoading) {
+    return (
+      <div className={classNames(className, wrapperClassName)}>
+        <Skeleton
+          containerClassName="pgn__card-footer-loader"
+          height={skeletonHeight}
+          width={skeletonWidth}
```

Comment:
[clarification] I'm not exactly sure a consumer should need to override the `Skeleton` widths in these loading states here and the other subcomponents. I could definitely see wanting to configure height, but I would assume all skeleton states (except maybe the logo image skeleton) should just take up 100% width of its container?

Was there a reason to add the option to configure the skeleton widths that I'm missing? 😄

Reply:
Yes, you are right. React Skeleton can indeed have the width of its container. I think that in horizontal custom elements  Edx, the width (or height) of the element is not always set, the width of the skeleton, which is designated as a percentage, needs to be calculated from something with its size parameters. It seems to me that it is better to provide the user with the most flexible implementation of his desires))

Reply:
Yeah, probably won't hurt to keep it. The only downside I guess is a slightly more complex props API for consumers to understand but that should be OK; it's just 1 additional prop.